python/rainbow/simulators/prox_soft_bodies/types.py

```diff
@@ -269,6 +269,7 @@ def __init__(self):
         This initializes an engine instance which holds all the configuration data of the world that is to
         be simulated.
         """
+        self.body_type = 'soft_body' 
```

Comment:
Please add a comment about what this member variable is used for.

Perhaps this is more a simulator_type that is "soft bodies"?

I guess this is used elsewhere in the framework to abstract over rigid and soft body simulators. To make code sharing easier?